File: src/minimal_demo/scripts/Pub.py
#!/usr/bin/env python 
import rospy
from std_msgs.msg import Float64

# Import other necessary libraries

class <Class name>():
    def __init__(self):
        rospy.init_node('<node name>', anonymous=True)
        self.pub = rospy.Publisher('<ROS topic name', <message type>, queue_size=10)
        self.rate = rospy.Rate(<Integer>) 
    # Define a subclass
    def <subClass name>(self):  
        # To publish the message
        self.pub.publish(<the value>)

def main():
    obj = <Class name>()
    # obj.<subclass name>
        

if __name__ == '__main__':
    try:
       main() 
    except rospy.ROSInterruptException:
        pass

File: src/minimal_demo/scripts/Pub.cpp
#include <ros/ros.h>
#include <std_msgs/Float64.h>
// Other important libraries

int main(int argc, char **argv)
{
  ros::init(argc, argv, "< node name>");
  ros::NodeHandle n;
  ros::Publisher joint1_pub;
  joint1_pub = n.advertise<std_msgs::Float64>("< ROS topic name", 10); 
  ros::Rate loop_rate(10);
  while (ros::ok())
  {
    std_msgs::Float64 <message name>;
    <message name>.data = 2.0;
    joint1_pub.publish(<message name>);
    ros::spinOnce();
    loop_rate.sleep();
  }
}
